crates/router/src/connector/globalpay/response.rs

```diff
@@ -156,7 +153,12 @@ pub struct Apm {
     /// The reference the payment method provider created for the transaction.
     pub provider_transaction_reference: Option<String>,
     /// URL to redirect the payer from the merchant's system to the payment method's system.
-    pub redirect_url: Option<Url>,
+    //1)paypal sends redirect_url as provider_redirect_url for require_customer_action
+    //2)bankredirects sends redirect_url as redirect_url for require_customer_action
+    //3)after completeauthorize in paypal it doesn't send redirect_url
+    //4)after customer action in bankredirects it sends empty string in redirect_url
+    #[serde(alias = "provider_redirect_url")]
+    pub redirect_url: Option<String>,
```

Comment:
Change this to Url type instead of string

Reply:
After customer action in bank redirects it sends an empty string in redirect_url which fails to parse as Url . So i have handled this case in transformers .If string is not empty , we parse it into Url .